buildtools/maven/pom.go

```diff
@@ -55,13 +56,17 @@ func PomFileGraph(target, dir string) (graph.Deps, error) {
 	if err != nil {
 		return graph.Deps{}, err
 	}
+
+	// Aggregate `dependencies` and `dependencyManagement` fields.
+	dependencyList := append(pom.Dependencies, pom.DependencyManagement...)
```

Comment:
I think you need to de-dup after this.

Reply:
...maybe, though maybe not

Reply:
There's a chance that pom files are only allowed to use one of these, in which case I won't need to do any appending. I'm going to look into that.